### PR TITLE
adds sound controls, soundwave and skeleton session page

### PR DESCRIPTION
**Pull request by TS-Stefan**

### CODE REVIEW

File: apps/koala-frontend/src/app/features/sessions/pages/session/session.page.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    this.sessionId = parseInt(this.route.snapshot.paramMap.get('sessionId') || '0');
+    this.sessionService.getOne(this.sessionId).subscribe((result) => {
+      this.session = {
+        ...result.data?.session,
+        media: result.data?.session.media,
+      };
+      if (this.session.media == undefined) {
+        this.messageService.add({
+          severity: 'info',
+          summary: 'Kein Audio File gefunden',
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/pages/session/session.page.ts:45]: Could you make these texts translatable

File: apps/koala-frontend/src/app/features/sessions/pages/session/session.page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { ApolloTestingController, ApolloTestingModule } from 'apollo-angular/testing';
+import { SessionsModule } from '../../sessions.module';
+
+import { SessionPage } from './session.page';
+
+describe('SessionCreateComponent', () => {
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/pages/session/session.page.spec.ts:8]: I guess we should this file as we did for the other components as long as we do not want to add component tests (which I cannot see here at the moment)

File: apps/koala-frontend/src/app/features/sessions/pages/session/session.page.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        </div>
+        <div class="sideBar" *ngIf="getAudioMetadata()">
+          <koala-app-audio-player
+            [metadata]="getAudioMetadata()"
+            (mediaEvent)="onMediaEvent($event)"
+          ></koala-app-audio-player>
+        </div>
+      </div>
+    </p-tabPanel>
+    <p-tabPanel header="Auswertung"> </p-tabPanel>
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/pages/session/session.page.html:16]: make header translatable

File: apps/koala-frontend/src/app/features/sessions/components/audio-player/audio-player.component.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<div [id]="mc.uuid"></div>
```
Comment (TS-Daniel16) [inline apps/koala-frontend/src/app/features/sessions/components/audio-player/audio-player.component.html:1]: What are these used for?

**Review by TS-Daniel16 — CHANGES_REQUESTED** [review]

File: apps/koala-frontend/src/app/features/sessions/services/media-control.service.ts [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+            container: `#${this.uuid}-timeline`,
+          }),
+          plugin,
+        ],
+      })
+    );
+    const w = this.waves.get(this.uuid);
+    if (w !== undefined) {
+      const audio = new Audio();
+      audio.src = URL.createObjectURL(audioBlob);
```
[the remark this answers is not in the document]
Comment (TS-Stefan) [reply apps/koala-frontend/src/app/features/sessions/services/media-control.service.ts:75]: I am throwing an error in the catch. So it should jump out of the `load` function.

File: apps/koala-frontend/src/app/features/sessions/components/audio-player/audio-player.component.html (hunk 4 above)
Comment (TS-Stefan) [reply apps/koala-frontend/src/app/features/sessions/components/audio-player/audio-player.component.html]: removed

**Pull request merged**